src/odoc/bin/main.ml

```diff
@@ -688,6 +688,7 @@ end = struct
           current_dir;
         }
     in
+    let directories = directories @ List.map ~f:snd lib_roots in
```

Comment:
I was about to say that this requires an update to the man for the `-L` option, but it was actually already saying that!

So whenever we can write `{!/libname/Module}` we can also write `{!Module}`.

Reply:
I wasn't sure whether adding the paths to the include path was the best approach, or whether we should do something else elsewhere?